src/ray/rpc/object_manager/object_manager_server.h

```diff
@@ -57,7 +57,7 @@ class ObjectManagerGrpcService : public GrpcService {
             service_, &ObjectManagerService::AsyncService::RequestPush, service_handler_,
             &ObjectManagerServiceHandler::HandlePushRequest, cq, main_service_));
     server_call_factories_and_concurrencies->emplace_back(std::move(push_call_factory),
-                                                          50);
+                                                          1);
```

Comment:
Curious what effect this has?

Reply:
This is the maximum number of outstanding rpcs allowed on this particular completion queue. For large objects, 1 rpc per completion queue is enough to saturate a 25Gbps network. Let me test for a variety of object sizes and then let you know.